### ISSUE TITLE
_physics_process is triggered even if space is inactive

### ISSUE THREAD

**Issue by capsizedmoose** [appsinacup/godot-rapier-physics#318]
**Describe the bug**

Even when the space is inactive, the _physics_process is triggered on Rigidbodies and CharacterControllers. 

**To Reproduce**

- Inactivate the physics space with PhysicsServer3D.space_set_active(space, false);
- add behaviour to _physics_process of a rigidbody or character controller to monitor that it's called

**Expected behavior**

I expected the _phyics_process to be called only when the physics should update (ie. while stepping through the physics automatically or manually)

**Environment:**
 - OS: Windows
 - Version: 0.8.8
 - Godot Version: v4.4.dev4
 - Type: godot-rapier-3d-single-enhanced-determinism

**Example project(zip)**

[RapierPhysicsProcessTest.zip](https://github.com/user-attachments/files/18080510/RapierPhysicsProcessTest.zip)

**Comment by Ughuuu** [comment]
I will check but if this also happens in godot, then this is probably how it should be. If you want a change you should open instead an issue on godot side actually.

**Comment by Ughuuu** [comment]
Right, it is. So instead, you should probably disable the physics_process callback on all the objects. Not much I can do here from the physics server.
